### PR TITLE
Feat/ Add style to new admin view 

### PR DESCRIPTION
**Pull request by dottyy**
This PR focuses on editing the design in the admin panel, including  the addition of new elements for improved functionality and aesthetics.

New Design Elements:

Added new design elements to the admin panel to support future features and functionalities.

### CODE REVIEW

**Comment by dottyy** [comment]
Hi @roguenet !.
Could you please review this PR when you have a moment? Your feedback is highly valuable to me.
Thanks in advance for your time.

File: src/admin/components/Actions/ListActionsCreate.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+const ListActionsCreate = (props: ListActionsCreateProps) => (
+  <TopToolbar>
+    <CreateButton className="filter-button-page-admin" />
+    <When condition={!!props.onExport}>
+      <Button className="button-page-admin" label="Export" startIcon={<DownloadIcon />} onClick={props.onExport} />
+    </When>
+  </TopToolbar>
+);
```
Comment (roguenet) [inline src/admin/components/Actions/ListActionsCreate.tsx:16]: This is fine as-is, but it might be easier to read with destructuring in the constructor:
```
const ListActionsCreate = ({ onExport }: ListActionsCreateProps) => {
 ...
}
```
(and then just use `onExport` instead of `props.onExport`

File: src/admin/components/App.tsx [hunk 2]
```diff
@@ -50,29 +42,29 @@ const App = () => {
...
+        icon={() => <Icon className="h-8 w-8" name={IconNames.PITCHES} />}
       />
       <Resource
         name={modules.fundingProgramme.ResourceName}
         list={modules.fundingProgramme.List}
         edit={modules.fundingProgramme.Edit}
         show={modules.fundingProgramme.Show}
         create={modules.fundingProgramme.Create}
-        icon={AttachMoneyIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.FUJDING_PROGRAMMES} />}
```
Comment (roguenet) [inline src/admin/components/App.tsx:67]: Oh, I left a comment on the other PR about this as well, but I notice now that it's just a mispelling. Please change to `FUNDING_PROGRAMMES`

File: src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts [hunk 3]
```diff
@@ -51,7 +50,7 @@ export default function useFormChanges(current: any, changed: any, steps: FormSt
               title: entry.title,
               type: entry.type,
               currentValue,
-              newValue: isEqual(currentValue, newValue) ? undefined : newValue
+              newValue: currentValue === newValue ? undefined : newValue
```
Comment (roguenet) [inline src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts:53]: Commented about this on the other PR: please revert this change, it will break existing functionality.

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/ComentarySection/ComentarySection.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      <CommentaryBox
+        name={authMe?.data.first_name}
+        lastName={authMe?.data.last_name}
+        refresh={refresh}
+        record={record}
+        entity={entity}
+      />
+      <When condition={viewCommentsList}>
+        <div className="max-h-[60vh] min-h-[10vh] grid-cols-[14%_20%_18%_15%_33%] overflow-auto">
+          {auditLogData !== null && auditLogData !== undefined ? (
```
Comment (roguenet) [inline src/admin/components/ResourceTabs/PolygonReviewTab/components/ComentarySection/ComentarySection.tsx:44]: This can be simplified to `auditLog != null`. When null checking I typically use `== null` or `!= null` to catch both `null` and `undefined`

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+      <ModalConfirm
+        title={`${titleStatus} Status Change`}
+        commentArea
+        menuLabel={""}
+        menu={menuOptionsMap[titleStatus]}
+        onClose={closeModal}
+        content={contentStatus}
+        onConfirm={async (text: any, opt) => {
+          const option = menuOptionsMap[titleStatus].find(option => option.value === opt[0]);
+          let response;
```
Comment (roguenet) [inline src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx:142]: Since this is never reassigned and only used inside the try block, please declare as a const on line 144

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+                title: "Success!"
+              });
+            }, 3000);
+          } catch (e) {
+            alert("The request encountered an issue, or the comment exceeds 255 characters.");
+            console.error(e);
+          } finally {
+            refresh();
+            reloadEntity();
+            closeModal;
```
Comment (roguenet) [inline src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx:175]: This looks like it's supposed to be a function call? `closeModal();`

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
```
Comment (roguenet) [inline src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx:222]: `closeModal()`

**Review by roguenet — COMMENTED** [review]
This PR has a lot of content from the other PR. In the future, when opening a second PR that includes contents of another branch, please base the PR on that branch so we can see specifically what's new in this branch.

File: src/admin/components/Actions/ListActionsCreate.tsx (hunk 1 above)
Comment (dottyy) [reply src/admin/components/Actions/ListActionsCreate.tsx:16]: Thanks for the suggestion! I updated the code to use destructuring in the constructor for better readability.

File: src/admin/components/App.tsx (hunk 2 above)
Comment (dottyy) [reply src/admin/components/App.tsx]: done

File: src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts (hunk 3 above)
Comment (dottyy) [reply src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts]: done

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/ComentarySection/ComentarySection.tsx (hunk 4 above)
Comment (dottyy) [reply src/admin/components/ResourceTabs/PolygonReviewTab/components/ComentarySection/ComentarySection.tsx]: Thanks for the suggestion! I simplified the check as you recommended.

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx (hunk 5 above)
Comment (dottyy) [reply src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx]: Thanks for the feedback! I've updated the code to declare the variable as a constant within the try block, as suggested.

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx (hunk 6 above)
Comment (dottyy) [reply src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx]: Thanks for pointing that out! I've updated the code to ensure closeModal is called as a function.

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx (hunk 7 above)
Comment (dottyy) [reply src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx]: Thanks for pointing that out! I've updated the code to ensure closeModal is called as a function.
